src/locale/locales/hu/messages.po

```diff
@@ -0,0 +1,7651 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Bluesky Social\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-11-05 16:01-0800\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Levente Pál Szíjártó <szijartoleventepal@gmail.com>\n"
+"Language-Team: Levente Pál Szíjártó – szijartoleventepal@gmail.com\n"
+"Language: hu\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.5\n"
+
+#: src/screens/Messages/components/ChatListItem.tsx:130
+msgid "(contains embedded content)"
+msgstr "(beágyazott tartalom)"
+
+#: src/view/com/modals/VerifyEmail.tsx:150
+msgid "(no email)"
+msgstr "(nincs megadott email-cím)"
+
+#: src/view/com/notifications/FeedItem.tsx:232
+#: src/view/com/notifications/FeedItem.tsx:327
+msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
+msgstr "{formattedCount} további"
+
+#: src/lib/hooks/useTimeAgo.ts:156
+msgid "{0, plural, one {# day} other {# days}}"
+msgstr "# nappal"
+
+#: src/lib/hooks/useTimeAgo.ts:146
+msgid "{0, plural, one {# hour} other {# hours}}"
+msgstr "# órával"
+
+#: src/components/moderation/LabelsOnMe.tsx:54
+msgid "{0, plural, one {# label has been placed on this account} other {# labels have been placed on this account}}"
+msgstr "# feljegyzést helyeztek erre a fiókra"
+
+#: src/components/moderation/LabelsOnMe.tsx:60
+msgid "{0, plural, one {# label has been placed on this content} other {# labels have been placed on this content}}"
+msgstr "# feljegyzést helyeztek erre a tartalomra"
+
+#: src/lib/hooks/useTimeAgo.ts:136
+msgid "{0, plural, one {# minute} other {# minutes}}"
+msgstr "# perccel"
+
+#: src/lib/hooks/useTimeAgo.ts:167
+msgid "{0, plural, one {# month} other {# months}}"
+msgstr "# hónappal"
+
+#: src/view/com/util/post-ctrls/RepostButton.tsx:71
+msgid "{0, plural, one {# repost} other {# reposts}}"
+msgstr "# megosztás"
+
+#: src/lib/hooks/useTimeAgo.ts:126
+msgid "{0, plural, one {# second} other {# seconds}}"
+msgstr "# másodperccel"
+
+#: src/components/ProfileHoverCard/index.web.tsx:398
+#: src/screens/Profile/Header/Metrics.tsx:23
+msgid "{0, plural, one {follower} other {followers}}"
+msgstr "követő"
+
+#: src/components/ProfileHoverCard/index.web.tsx:402
+#: src/screens/Profile/Header/Metrics.tsx:27
+msgid "{0, plural, one {following} other {following}}"
+msgstr "követett"
+
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:312
+msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
+msgstr "Tetszik (# kedvelés)"
+
+#: src/view/com/post-thread/PostThreadItem.tsx:434
+msgid "{0, plural, one {like} other {likes}}"
+msgstr "kedvelés"
+
+#: src/components/FeedCard.tsx:213 src/view/com/feeds/FeedSourceCard.tsx:300
+msgid "{0, plural, one {Liked by # user} other {Liked by # users}}"
+msgstr "# felhasználó kedveli"
```

Comment:
```suggestion
msgstr "{0, plural, other {# felhasználó kedveli}}"
```

Reply:
Maybe expressing the plural in this way will work, it works well in the Chinese translation.

Reply:
I have pushed a commit right before your answer with a different possible solution. Would you might checking if it works, please?